README.md

```diff
@@ -55,14 +65,51 @@ vim .env
 ### Run the worker Steps
 
 1. Run the worker
+
 ```sh
 docker compose up -d
 ```
 
+### Run the worker from source
+
+1. Clone this repo
+
+```sh
+git clone https://github.com/Lagrange-Labs/lgn-coprocessor.git
+cd  lgn-coprocessor
+```
+
+2. Check your Rust toolchain
+
+```sh
+rustup toolchain list
+nightly-2024-12-03... (It is recommended)
+```
+
+2. Update the worker configuration
+
+```sh
+cat lgn-worker/src/config/default.toml
+```
+
+3. Update the system maximum stack size to unlimited
+
+```sh
+ulimit -s unlimited
```

Comment:
@delehef any blockers on this ? @silathdiir doesn't it impact performance ?

Reply:
>  any blockers on this ?

AFAIK, the only issue is that accidental infinite recursion would blow up the whole memory instead of early killing the faulty program, so I don't think that enabled unlimited stack in this context a big risk.